internal/cmd/flags.go

```diff
@@ -41,13 +41,20 @@ var (
 		fs.StringVar(p, PflagSignatureFormat.Name, defaultSignatureFormat, PflagSignatureFormat.Usage)
 	}
 
-	PflagTimestamp = &pflag.Flag{
-		Name:      "timestamp",
-		Shorthand: "t",
-		Usage:     "timestamp the signed signature via the remote TSA",
+	PflagID = &pflag.Flag{
+		Name:  "id",
+		Usage: "key id (required if --plugin is set). This is mutually exclusive with the --key flag",
 	}
-	SetPflagTimestamp = func(fs *pflag.FlagSet, p *string) {
-		fs.StringVarP(p, PflagTimestamp.Name, PflagTimestamp.Shorthand, "", PflagTimestamp.Usage)
+	SetPflagID = func(fs *pflag.FlagSet, p *string) {
+		fs.StringVar(p, PflagID.Name, "", PflagID.Usage)
+	}
+
+	PflagPlugin = &pflag.Flag{
+		Name:  "plugin",
+		Usage: "signing plugin name. This is mutually exclusive with the --key flag",
```

Comment:
```suggestion
		Usage: "signing plugin name (requried if --id is set). This is mutually exclusive with the --key flag",
```

Reply:
I think that makes sense. Typo in "required". It should also probably be added for --id. I'll make a quick commit to add this

Reply:
Resolved in newest commit